### DISCUSSION TITLE
Shipping confirmation and conditional formatting.

### DISCUSSION THREAD

**Discussion by hugopvl** [directus/directus#6278] [Archive]
Hey guys, I needed to do the following, I have a column with the date of shipment, when the shipment is made I need some way to confirm that shipment and change the color date. for example green.

Is there any idea how to implement this situation in directus, has anyone done something similar?

Thanks for listening.

**Comment by philleepflorence** [comment]
Looks like you want to update the `color date` after the `date of shipment`?
Do you want this to be an automated/scheduled task (cron job), assuming the `date of shipment` is filled in advance?

Just in case: a [Custom API Interval Hook](https://docs.directus.io/guides/api-hooks/#interval-cron) may be a solution.

> when the shipment is made I need some way to confirm that shipment...  

Are you confirming the shipment date or actual shipment?

**Reply by hugopvl** [reply]
I have an estimated shipping date, then I need to confirm the actual shipping date and change the date color to green..

**Reply by philleepflorence** [reply]
How are you confirming the `actual shipping date`, is it coming from a service like [Postage Tracker](https://rapidapi.com/FINDBINNUMBERS/api/postage-tracker/), or is it a manual confirmation?

Is this an e-commerce project?

**Reply by hugopvl** [reply]
I'm not doing the confirmation of the date yet, but it will have to be manual because it's an internal process.

it's not an e-commerce project,  it's an approval system/planner

**Reply by hugopvl** [reply]
Hi,

The date confirmation is done, now is it possible to change the date color with api hook?

**Reply by benhaynes** [reply]
Our date interface/display doesn't support text color as of now. You could duplicate it as a custom extension to add this feature, or use a separate field with the "labels" display.

**Reply by hugopvl** [reply]
ok, solved this with dot labels 

thanks.
